pkg/utils/helm/helm.go

```diff
@@ -132,7 +132,11 @@ func GetChartVersion(chart string) (version string, err error) {
 		"|", "grep", "version:"}
 	log.V(1).Info("Exec bash -c", "args", args)
 
-	cmd := exec.Command("bash", "-c", strings.Join(args, " "))
+	// cmd := exec.Command("bash", "-c", strings.Join(args, " "))
+	cmd, err := utils.PipeCommand("bash", "-c", strings.Join(args, " "))
```

Comment:
It seems that this is unnecessary to use pipe. We can use `ddc-helm inspect chart <chart>` and scan the returned string. I would like to change this in a next PR.